apps/client/src/locales/messages.es.xlf

```diff
@@ -0,0 +1,2651 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+  <file source-language="en" datatype="plaintext" original="ng2.template">
+    <body>
+      <trans-unit id="e19fcf996343543e13789f17b550ab0c08124b5c" datatype="html">
+        <source>Create Account</source>
+		<target state="translated">Crear una cuenta</target>
```

Comment:
Can you please replace the tabs with double spaces that it matches the style?

Reply:
I did it, but I am not sure if it is better.

Reply:
Sorry to bother you, it is still not correct:

<img width="1745" alt="messages-xlf" src="https://user-images.githubusercontent.com/4159106/192160329-baae6be5-6e4c-48c3-9c64-1072ccc926c0.png">

You can see it in [Files changed](https://github.com/ghostfolio/ghostfolio/pull/1292/files): Click _Load diff_

Reply:
I am not sure what level of indentation is desired. I took the <source> level and added two spaces.